tests/inspection_tests.py

```diff
@@ -202,7 +224,8 @@ def get_service_health(service_host):
                 response_headers = res.getheaders()
                 print("RESPONSE HEADERS:")
                 print(response_headers)
-                extract_zip(file_path, folder_path)
+                if "blob_upload" not in inspection_test or inspection_test["blob_upload"] == False:
```

Comment:
Can we HTTP GET the blob that was created with the blob_upload SaS url we have and validate the contents here for success cases?

Reply:
Added a way for unit tests to download the resulting blob and perform validation against expected file lists.